packages/ember-data/lib/system/model/states.js

```diff
@@ -343,7 +345,7 @@ var DirtyState = {
 
     rolledBack: function(record) {
       get(record, 'errors').clear();
-      record.triggerLater('ready');
+      record.transitionTo('loaded.saved');
```

Comment:
@pangratz I fix it this way but I still think that this should be done in some better way but my solution might be useful for anybody who needs to fix this right know. We should always refactor and change logic after final will be made.